CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ You can find our backwards-compatibility policy [here](https://github.com/hynek/
 - Support for Python 3.14 and Python 3.13.4.
   [#723](https://github.com/hynek/structlog/pull/723)
 
+### Fixed
+- `structlog.processors.ExceptionPrettyPrinter` now uses *exception_formatter* instead of the default formatter if passed.
```

Comment:
```suggestion

- `structlog.processors.ExceptionPrettyPrinter` now respects the *exception_formatter* arguments instead of always using the default formatter.
```